controllers/user.controller.js

```diff
@@ -0,0 +1,93 @@
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const User = require("../models/user.model");
+const authConfig = require("../configs/auth.config");
+const constants = require("../utils/constants")
+
+exports.signup = async(req,res) => {
```

Comment:
1. This code should be inside auth.controller.js
2. This file should contain the code for updating the user - so that's admin owner can be approved